runtime/string.inl

```diff
@@ -344,6 +344,18 @@ bool string::ends_with(const string &other) const noexcept {
   return other.size() > size() ? false : !memcmp(c_str() + (size() - other.size()), other.c_str(), other.size());
 }
 
+bool string::contains(const string &other) const noexcept {
+  if (other.size() > size())
+    return false;
```

Comment:
It's a good practice to always use `{}` for statements like `if` and `for`.
Please use it here and in other places.

Reply:
Fixed :)